lib/fetch/file.js

```diff
@@ -278,7 +278,9 @@ function processBlobParts (parts, options) {
       if (!element.buffer) { // ArrayBuffer
         bytes.push(new Uint8Array(element))
       } else {
-        bytes.push(element.buffer)
+        bytes.push(
+          new Uint8Array(element.buffer, element.byteOffset, element.byteLength)
+        )
       }
```

Comment:
Actually this is still wrong. We need to make a copy, not just a new reference. Could you fix that?

Reply:
@KhafraDev is this something you could look into? The copy part.

Reply:
Will do. On vacation for the next 6 days but I'll see if I can get to it